packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-dialog.tsx

```diff
@@ -32,18 +35,27 @@ const ContractTypeDialog = ({
     is_open,
     item,
     selected,
+    title,
     onBackButtonClick,
     onCategoryClick,
     onChangeInput,
     onClose,
     onSearchBlur,
     show_loading,
+    learn_more_banner,
+    hide_back_button,
 }: React.PropsWithChildren<TContractTypeDialogProps>) => {
     const current_mobile_title = is_info_dialog_open ? (
-        <Header title={item?.text || ''} onClickGoBack={onBackButtonClick} text_size='xs' />
+        <Header
+            title={localize(title || item?.text || '')}
```

Comment:
localize won't work with variables

Reply:
Will remove `localize() `from here and wrap `title` with `<Localize/>` inside of `<header>`

Reply:
we just need to make sure that title or item?.text is already localized when it's used here. :)

Reply:
it was already localized in packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx on line 276. Will revert this changes